docs/declarative-customization/view-calendar-formatting.md

```diff
@@ -16,12 +16,18 @@ Groups the command bar customization options. For details on `commandBarProps`,
 ## See also
 - [Command bar customization syntax reference](./view-commandbar-formatting.md)
 
+### List setup before creating a calendar view and applying a JSON formatting
+Before creating a new calendar view, a list should include all the required columns to be used in any JSON formatting.
+After creating a calendar view, if more columns are required, go to the [List Settings page](https://support.microsoft.com/en-us/office/edit-list-settings-4d35793b-246e-42a3-990c-563a83795b7f) to create and add them to a calendar view.
```

Comment:
Never include locale in MSFT URLs... let hosting engine route accordingly. Only exception is when there's no default page to route to.